docs/config.md

```diff
@@ -268,10 +268,10 @@ You can configure the set of immutable commits via
 `revset-aliases."immutable_heads()"`. The default set of immutable heads is
 `builtin_immutable_heads()`, which in turn is defined as
 `present(trunk()) | tags() | untracked_remote_bookmarks()`. For example, to
-also consider the `maint@origin` bookmark immutable:
+also consider the `main@origin` bookmark immutable:
```

Comment:
I thought this would mean the "maint"-enance branch because `main` is usually included in `trunk()`. If that's the case, `release@origin` might be a better example. @martinvonz ?

Reply:
Yes, it was meant to be the maintenance branch. I'll send a PR to change it to `release@origin`.